dependencies/maven/artifacts.bzl

```diff
@@ -30,11 +30,14 @@ artifacts = [
     "org.jetbrains.compose.ui:ui-desktop",
     "org.jetbrains.compose.ui:ui-geometry-desktop",
     "org.jetbrains.compose.ui:ui-graphics-desktop",
+    "org.jetbrains.compose.ui:ui-test-desktop",
+    "org.jetbrains.compose.ui:ui-test-junit4-desktop",
     "org.jetbrains.compose.ui:ui-text-desktop",
     "org.jetbrains.compose.ui:ui-unit-desktop",
     "org.jetbrains.kotlin:kotlin-test",
     "org.jetbrains.kotlinx:kotlinx-coroutines-core",
     "org.jetbrains.kotlinx:kotlinx-coroutines-core-jvm",
+    "org.jetbrains.kotlinx:kotlinx-coroutines-test",
```

Comment:
We require a few dependencies in order for the test infrastructure to work. 

The first two are provided by jetbrains as a part of compose-jb for the purposes of testing desktop applications. 

The final is another provided by jetbrains but more broadly for the purposes of testing code that requires the use of coroutines. As Studio is a multi-threaded application, we need a multi-threaded testing environment to block and await at certain points.